<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>99</width>
       <height>79</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="username_t">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="password_t">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="login_b">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>user</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forget_b">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>forget</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>pass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rigister_b">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>rigister</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
